components/Nav/NavListItem/NavListItem.js

```diff
@@ -41,10 +41,19 @@ export default class NavListItem extends Component {
     this.setState(previousState => ({ areSublinksVisible: !previousState.areSublinksVisible }));
   };
 
+  handleKey = (event, isLastSublink) => {
+    if (event.key === 'Tab') {
```

Comment:
I'd love if all of this conditions were made into variables, and if the final variable was just `shouldHideSublinks`.

Something like:

```javascript
handleKeyDown = (event, indexKeyedOn) => {
  const { sublinks } = this.props;

  const didHitTab = event.key === 'Tab';
  const didTabForward = didHitTab && !event.shiftKey;
  const didTabBackward = didHitTab && event.shiftKey;

  const lastSublinkIndex = sublinks.length - 1;
  const isLastSublink = lastSublinkIndex === indexKeyedOn
  const isFirstSublink = indexKeyedOn === 0;

  const shouldHideSublinks = (isLastSublink && didTabForward) || (isFirstSublink && didTabBackward);

  if (shouldHideSublinks) {
    this.hideSublinks();
  }
}
```